genrl/deep/agents/a2c/a2c.py

```diff
@@ -137,12 +144,12 @@ def create_model(self) -> None:
 
         self.critic_optimizer = opt.Adam(self.ac.critic.parameters(), lr=self.lr_critic)
 
-        self.traj_reward = []
-        self.actor_hist = torch.Tensor().to(self.device)
-        self.critic_hist = torch.Tensor().to(self.device)
-
-        self.actor_loss_hist = torch.Tensor().to(self.device)
-        self.critic_loss_hist = torch.Tensor().to(self.device)
+        self.rollout = RolloutBuffer(
+            2048,
+            self.env.observation_space,
+            self.env.action_space,
+            n_envs=self.env.n_envs,
+        )
```

Comment:
Is 2048 a standard number or can we make that more generic like replay_size?

Reply:
Yeah, we can change this.